regression/regression_var.sh

```diff
@@ -33,7 +33,7 @@ if [[ -d /glade ]]; then # Cheyenne
   export machine="Cheyenne"
 elif [[ -d /scratch1 ]]; then # Hera
   export machine="Hera"
-elif [[ -d /jetmon ]]; then # Jet
+elif [[ -d /mnt/lfs4 ]]; then # Jet
```

Comment:
"/jetmon" would be more general. Although it does not happen often, but there are times when /lfs4 may not be mounted.

Reply:
At the time that I was testing this, /jetmon was not available.  I will add additional jet-specific checks in case /mnt/lfs4 is not mounted.

Reply:
Updated to include /jetmon and /mnt/lfs1.